src/core/lib/slice/slice_internal.h

```diff
@@ -314,17 +315,36 @@ grpc_slice grpc_slice_from_moved_string(grpc_core::UniquePtr<char> p);
 // 0. All other slices will return the size of the allocated chars.
 size_t grpc_slice_memory_usage(grpc_slice s);
 
-inline grpc_slice grpc_slice_from_static_buffer_internal(const void* s,
-                                                         size_t len) {
-  grpc_slice slice;
-  slice.refcount = &grpc_core::kNoopRefcount;
-  slice.data.refcounted.bytes = (uint8_t*)s;
-  slice.data.refcounted.length = len;
-  return slice;
+inline grpc_core::StaticSlice grpc_slice_from_static_buffer_internal(
+    const void* s, size_t len) {
+  return grpc_core::StaticSlice(
+      &grpc_core::kNoopRefcount, len,
+      reinterpret_cast<uint8_t*>(const_cast<void*>(s)));
+  //  slice.refcount = &grpc_core::kNoopRefcount;
```

Comment:
The commented-out code should probably be removed

Reply:
Removed.